File: msgraph-mail/go/utilities/users/followup_flag.go
package users
import (
    "github.com/microsoft/kiota/abstractions/go/serialization"
)
type FollowupFlag struct {
    additionalData Map<String, Object>;
    completedDateTime DateTimeTimeZone;
    dueDateTime DateTimeTimeZone;
    flagStatus FollowupFlagStatus;
    startDateTime DateTimeTimeZone;
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Models/Microsoft/Graph/FollowupFlag.swift
extension Swiftconsoleapp.Models.Microsoft.Graph {
public class FollowupFlag : IAdditionalDataHolder {
public var additionalData: IDictionary<string, object>
public var completedDateTime: DateTimeTimeZone
public var dueDateTime: DateTimeTimeZone
public var flagStatus: FollowupFlagStatus?
public var startDateTime: DateTimeTimeZone
}
}
